backend/src/common/enums/app/env.enum.ts

```diff
@@ -31,6 +31,16 @@ const ENV = {
   SENTRY: {
     DSN: SENTRY_DSN,
   },
+  MAIL: {
+    API_URL: 'http://localhost:3000/',
+    ACTIVATE_URL: 'http://localhost:3001/api/v1/activate/',
+    SUCCESS_URL: 'http://localhost:3000/mail-success',
+    FAILED_URL: 'http://localhost:3000/mail-failed',
+    SMTP_HOST: 'smtp.gmail.com',
+    SMTP_PORT: 587,
+    SMTP_USER: 'autoline.binary@gmail.com',
+    SMTP_PASSWORD: 'pyvwijwvexspvzfi',
```

Comment:
Are you guys serious? 
### NEVER PROVIDE CREDENTIALS IN CODE!!!
Why did not use .env?

Reply:
I agree, fixed it. I left it to Serhii who does email verification task.